Apps/W1/Shopify/app/src/Companies/Codeunits/ShpfyCompanyMapping.Codeunit.al

```diff
@@ -22,57 +20,14 @@ codeunit 30303 "Shpfy Company Mapping"
 
     internal procedure FindMapping(var ShopifyCompany: Record "Shpfy Company"; var TempShopifyCustomer: Record "Shpfy Customer" temporary): Boolean;
     var
-        Customer: Record Customer;
-        ShopifyCustomer: Record "Shpfy Customer";
-        CustomerMapping: Codeunit "Shpfy Customer Mapping";
-        PhoneFilter: Text;
+        ShpfyCompByEmailPhone: Codeunit "Shpfy Comp. By Email/Phone";
```

Comment:
Please fix the variable naming. 

If same name exists in Base Application then Shopify prefix, otherwise no prefix.

Example:

ShopifyCustomer: Record "Shpfy Customer" -> customer exists in Base Application 
Catalog: Record "Shpfy Catalog" -> catalog does not exist in Base Application